packages/atomic/cypress/support/commands.ts

```diff
@@ -29,5 +32,19 @@ Cypress.Commands.add('getTextOfAllElements', (selector: string) => {
   });
 });
 
+interface CypressRequest {
+  xhr: {url: string};
+}
+
+// Only possible to filter on the url
+Cypress.Commands.add('shouldBeCalled', (urlPart, timesCalled) => {
```

Comment:
Neat!

Reply:
Thanks, but I was very disappointed that cypress did not provide anything clean OOTB for checking that a request was *not* made